onnxruntime/test/onnx/main.cc

```diff
@@ -784,14 +850,6 @@ select from 'TF8', 'TF16', 'UINT8', 'FLOAT', 'ITENSOR'. \n)");
     {"sequence_insert_at_back", "onnx currently not supporting loading segment", {}},
     {"sequence_insert_at_front", "onnx currently not supporting loading segment", {}},
     {"loop13_seq", "ORT api does not currently support creating empty sequences (needed for this test)", {}},
-    {"cast_FLOAT_to_BFLOAT16", "onnx generate bfloat tensor as uint16 type", {}},
```

Comment:
I see these got moved to `immutable_broken_tests` above. as there is a comment asking politely not to change that list, maybe leave them here?

Reply:
Sorry, I saw it, I forgot it. My screen is too small to see the whole list.